devtools/cli/src/main/java/io/quarkus/cli/RegistryListCommand.java

```diff
@@ -19,14 +20,15 @@
     public Integer call() throws Exception {
 
         registryClient.refreshRegistryCache(output);
+        final RegistriesConfig config = RegistriesConfigLocator.resolveConfig();
 
         output.info("Available Quarkus extension registries:");
```

Comment:
If it is including disabled registries, maybe it shouldn't start with `Available`? That was the point of this method - to show what's enabled.

Reply:
I kind of still think it's a good idea, btw.